docs/performance.md

```diff
@@ -28,16 +28,19 @@ as of [commit `fe7687c`](https://github.com/transparency-dev/tesseract/commit/fe
 The table below shows the measured performance over 12 hours in each instance type:
 
 | Instance Type                        | Cloud Spanner | Write QPS |
-| ------------------------------------ | ------------- | --------- |
+| ------------------------------------ | ------------- | --------: |
 | e2-micro (2 vCPUs, 1 GB Memory)      | 100 PUs       | 60        |
 | e2-medium (2 vCPUs, 4 GB Memory)     | 100 PUs       | 250       |
 | e2-standard-2 (2 vCPUs, 8 GB Memory) | 100 PUs       | 600       |
 
 | Managed Instance Group Type           | Instances | Cloud Spanner | Write QPS |
-| ------------------------------------- | --------- | ------------- | --------- |
+| ------------------------------------- | :-------: | ------------- | --------: |
+| n2-standard-4 (4 vCPUs, 16 GB Memory) | 1         | 100 PUs       | 1000      |
```

Comment:
This makes me wonder what 1 instance with 200 PUs will do - I'd expect it to be very similar to the 3 instance with 200 PUs below.

Reply:
Here is the Cloud Spanner CPU utilization graph from 200 PUs to 100 PUs.

200 PUs: 2:55 PM - 3:55 PM
100 PUs: 3:55 PM - 4:55 PM

<img width="1669" height="331" alt="image" src="https://github.com/user-attachments/assets/48595b28-1a96-4c80-8bcc-2861ab72dbec" />

Reply:
I chatted with @AlCutter on this yesterday. We suspected the bottleneck came from the default antispam pushback threshold. I've added the `pushback_max_antispam_lag` flag in https://github.com/transparency-dev/tesseract/pull/477 for another round of performance test.